src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,76 @@
+import facebook from "../../images/facebook.svg";
+import instagram from "../../images/instagram.svg";
+import linkedin from "../../images/linkedin.svg";
+
+export const Footer = () => {
+  return (
+    <footer className="flex w-full flex-col bg-wheat p-[80px] gap-[40px] sm:flex-col sm:p-[16px] sm:gap-[26px]">
+      <div className="flex flex-row gap-[287px] sm:gap-[26px] sm:flex-col">
+        <div className="flex flex-col gap-[40px] sm:gap-[26px]">
+          <div className="flex flex-col gap-[24px]">
+            <p className="font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:font-text sm:font-normal sm:leading-[26px] sm:tracking-[-0.02em] sm:text-center ">
+              Expert Event Solutions
+            </p>
+            <p
+              className="font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:font-text sm:font-normal sm:leading-[26px] sm:tracking-[-0.02em] sm:text-center  cursor-pointer"
+              onClick={() => console.log("clicked address")}
+            >
+              260 Daytona Blvd, Daytona <br /> Beach, Florida 32114, United{" "}
+              <br /> States
+            </p>
+            <p
+              className="font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:font-text sm:font-normal sm:leading-[26px] sm:tracking-[-0.02em] sm:text-center  cursor-pointer"
+              onClick={() => console.log("clicked email")}
+            >
+              Info@experteventsolutions.com
+            </p>
+          </div>
+        </div>
+
+        <div className="flex flex-col gap-[24px]">
+          <p
+            className="font-text text-paragraph-sm sm:hidden"
+            onClick={() => console.log("clicked contacts")}
+          >
+            Contacts
+          </p>
+          <p
+            className="font-text text-paragraph-sm sm:hidden"
+            onClick={() => console.log("clicked rental conditions")}
+          >
+            Rental Conditions
+          </p>
+
+          <div className="flex flex-wrap flex-col gap-[10px] sm:content-center">
+            <p className="font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:font-text sm:font-normal sm:leading-[26px] sm:tracking-[-0.02em] sm:text-center ">
+              Social Media
+            </p>
+            <div className="flex flex-row gap-[10px] cursor-pointer">
+              <img
```

Comment:
Hi josh I noticed that there aren't any links to the socials yet, would we be adding these later on since they have not given them yet?

Reply:
That would be something that is added later, after being provided with the information. I might bring that up to Tomer in the tuesday sync.